sdks/java/core/src/main/java/org/apache/beam/sdk/util/IOChannelUtils.java

```diff
@@ -52,6 +58,7 @@
    * <p>For example, when reading from "gs://bucket/path", the scheme "gs" is
    * used to lookup the appropriate factory.
    */
+  @VisibleForTesting
   public static void setIOFactory(String scheme, IOChannelFactory factory) {
```

Comment:
needs to be public for now. BigQueryIOTest uses it to register a mock io factory.